src/routing/table.rs

```diff
@@ -39,25 +40,40 @@ enum FindNodeResult {
     Nodes(Vec<NodeInfo>),
 }
 
-struct RoutingTable;
+
+/// Routing table
+struct RoutingTable {
+    nodes: std::collections::HashMap,
```

Comment:
You don't need the full prefix. You could just use `std::collections::HashMap` because you've `use`'d it on line 4.